test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -769,20 +761,17 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $key = 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion\'
         $regValue = (Get-ItemProperty -Path $key -Name $propertyName -ErrorAction SilentlyContinue).$propertyName
 
-        if ($propertyName -eq "InstallDate")
+        if ($null -eq $regValue)
         {
-            # more complicated case: InstallDate
-            if ($regValue)
-            {
-                return Get-UnixSecondsToDateTime $regValue
-            }
+            return $null
         }
-        else
+
+        if ($propertyName -eq "InstallDate")
         {
-            return $regValue
+            return [DateTimeOffset]::FromUnixTimeSeconds($regValue).DateTime
         }
 
-        return $null
+        return $regValue
```

Comment:
Please undo unrelated refactoring.

Reply:
It is correctness issue, `($regValue)` evaluates to `false` when the value is `0`.

Reply:
@xtqqczze Please don't change test code - we should see it works after code change.